docs/people-and-groups/saml-okta.md

```diff
@@ -6,26 +6,64 @@ title: SAML with Okta
+
+![SAML form](images/saml-form.png)
+
+You'll need to use the information in this form to set up SAML in Okta.
+
+## Set up SAML in Okta
+
+Before configuring SAML authentication in Metabase, you'll need to create and configure a new SAML app integration in Okta.
+
+### Create an app integration
+
+From the Okta **Admin** console, [create a new SAML app integration][okta-saml-docs] to use with Metabase.
+
+### Configure Okta SAML settings
+
+To configure Okta app integration with Metabase, you'll need to use the information in Metabase **Admin panel** > **Authentication** > **SAML**.
+
+#### General settings
+
+| Okta SAML                       | Metabase SAML                                                                                                                               |
+| ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Single sign-on URL**          | **URL the IdP should redirect to**. This is your Metabase [Site URL][site-url] -- it should start with `https://` and end with `/auth/sso`. |
+| **Audience URI (SP Entity ID)** | **SAML Application Name** ("Metabase" by default)                                                                                           |
+
+#### Attribute statements
+
+Metabase requires you to set some attributes in Okta:
+
+- email address
+- first name (given name)
+- last name (surname)
+
+These attributes will be passed from Okta to Metabase during authentication to automatically log people into their Metabase accounts.
+
+In **Attribute statements (optional)** section of the Okta application SAML setting, create attribute statements for the email, first name, and last name.
 
 | Name                                                                 | Value          |
 | -------------------------------------------------------------------- | -------------- |
-| `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`    | user.firstName |
 | `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress` | user.email     |
+| `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`    | user.firstName |
 | `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`      | user.lastName  |
 
-**End-users should not be able to edit the email address attribute**. Your IdP will pass the email address attribute to Metabase in order to log people into their Metabase accounts (or to create an account on the first login). If a person can change the email address attribute, they'll potentially be able to access Metabase accounts other than their own.
+The names of attribute statement in Okta should match the attribute names in Metabase. If you use non-default attribute names in Okta app configuration, you will also need to change the names for the attribute fields in Metabase in **Admin panel** > **Authentication** > **SAML**
```

Comment:
Should we mention case sensitivity? And is non-default the same as "custom"? What about additional?

```suggestion
The names of attribute statement in Okta should match the attribute names in Metabase (names are case sensitive). If you want to use custom attribute names in your Okta app configuration, you'll also need to change the names for the attribute fields in Metabase: **Admin panel** > **Authentication** > **SAML**.
```

Reply:
"custom attribute names" had the ambiguity whether it's `(custom attribute) names` or `custom (attribute names)` because there's a concept of a custom attribute (and it's even something that we reference later in the article). 

But in our case it's about using `(custom name)` for an attribute statement (and those attribute statements check regular, non-custom attributes).

There were customers who used different (from MB-provided default) names in Okta and had problems. They just needed to copy paste the same names back into MB